docs/deployment.rst

```diff
@@ -103,12 +103,27 @@ The |crest| does not require any coordination between instances, so you can easi
 deployment up or down. The only requirement for multiple instances is that you set a unique
 ``id`` for each instance.
 
-If you run more than one instance of the proxy you should provide some load balancing mechanism.
+If you run more than one instance of |crest| you should provide some load balancing mechanism.
 The simplest approaches use round-robin DNS or a discovery service to select one instance per
 application process at startup, sending all traffic to that instance. You can also use an HTTP load
 balancer, but individual instances must still be addressable to support the absolute URLs
-returned for use in consumer read and offset commit operations.
+returned for use in consumer read and offset commit operations. 
 
+High Availability
+~~~~~~~~~~~~~~~~~
+
+When deploying multiple |crest| instances it is important that your consumer client handle exceptions returned
+from a failed consumer instance. Following an exception your client should attempt to create a new 
+consumer instance using the load balancer address in order to swtich to an active |crest| instance. 
+As noted above the client should use the absolute URL returned after creating a consuer instance for 
+further communication. Producing messages through multiple |crest| instances doesn't require this level 
+of exception handling. 
+
+In most cases your client should attempt to delete a consumer instance before exiting. 
```

Comment:
This statement begs the question "what about in other cases"? if we don't know answer, might consider: 

```
Your client will attempt to delete a consumer instance before exiting.
```

Reply:
Great point, new commit to resolve.